### ISSUE TITLE
Introduction to used technologies

### ISSUE THREAD

**Issue by PhlppKrmr** [amosproj/amos2023ss04-kubernetes-inventory-taker#66]
## User story
1. As a developer
2. I want / need to get comfortable with the used technologies
3. So that I can productively work with them in the upcoming sprints

## Acceptance criteria
- [ ] Learn Golang
- [ ] Learn about Kubernetes
- [ ] Learn about used frontend framework

## Linked Issues
- **Blocked by:**
- **Related to:**
- **Implements**

## Additional information
- none

## Definition of done (DoD)
* Colours match with [color scheme](https://amos-p4.atlassian.net/l/cp/g5VzjL88)

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
